apps/worker/src/monitor/status.monitor.module.ts

```diff
@@ -0,0 +1,82 @@
+/*
+https://docs.nestjs.com/modules
+*/
+
+import { BullModule } from '@nestjs/bullmq';
+import { Module } from '@nestjs/common';
+import { EventEmitterModule } from '@nestjs/event-emitter';
+import { RedisModule } from '@liaoliaots/nestjs-redis';
+import { StatusMonitoringService } from './status.monitor.service';
+import { ConfigModule } from '../../../api/src/config/config.module';
+import { ConfigService } from '../../../api/src/config/config.service';
+import { BlockchainModule } from '../blockchain/blockchain.module';
+import { QueueConstants } from '../../../../libs/common/src';
+
+@Module({
+  imports: [
+    ConfigModule,
+    BlockchainModule,
+    EventEmitterModule,
+    RedisModule.forRootAsync(
+      {
+        imports: [ConfigModule],
+        useFactory: (configService: ConfigService) => ({
+          config: [{ url: configService.redisUrl.toString() }],
+        }),
+        inject: [ConfigService],
+      },
+      true, // isGlobal
+    ),
+    BullModule.forRootAsync({
+      imports: [ConfigModule],
+      useFactory: (configService: ConfigService) => {
+        // Note: BullMQ doesn't honor a URL for the Redis connection, and
+        // JS URL doesn't parse 'redis://' as a valid protocol, so we fool
+        // it by changing the URL to use 'http://' in order to parse out
+        // the host, port, username, password, etc.
+        // We could pass REDIS_HOST, REDIS_PORT, etc, in the environment, but
+        // trying to keep the # of environment variables from proliferating
+        const url = new URL(configService.redisUrl.toString().replace(/^redis[s]*/, 'http'));
```

Comment:
nit: I see this part being duplicated everywhere. I think we can refactor it in future.